src/main/java/org/opentripplanner/util/NonLocalizedString.java

```diff
@@ -22,6 +23,16 @@ public NonLocalizedString(@Nonnull String name) {
         this.name = name;
     }
 
+    /**
+     * Check if name is non-null and returns an instance of {@link NonLocalizedString}, otherwise
+     * returns null.
+     */
+    @Nullable
+    public static NonLocalizedString ofNullable(@Nullable String name){
+        if(name == null) return null;
```

Comment:
Is it the standard in OTP to have one liner If statements - I dont believe I have seen this before

Reply:
It is, but there needs to always be braces around the statement
```suggestion
        if(name == null) { return null; }
```